engine/flink/generic/src/main/resources/defaultModelConfig.conf

```diff
@@ -7,21 +7,12 @@
 
   timeout: 10s
   checkpointConfig {
-      checkpointInterval: 10m
+    checkpointInterval: 10m
   }
   rocksDB {
     enable: true
   }
 
-  kafka {
```

Comment:
Shouldn't be copied to `dev-application.conf` and to `src/universal/conf/application.conf`?

Reply:
it should... I also wonder about consumerGroupNamingStrategy - shouldn't it be the default?

Reply:
done